plugins/event_source/aws_cloudtrail.py

```diff
@@ -0,0 +1,109 @@
+"""
+aws_cloudtrail.py
+
+An ansible-rulebook event source module for getting events from an AWS CloudTrail
+
+Arguments:
+    connection        - Parameters used to create AWS session
```

Comment:
can you added supported parameters? https://aiobotocore.readthedocs.io/en/latest/tutorial.html